ddocs/medic/views/docs_by_replication_key/map.js

```diff
@@ -67,8 +67,9 @@ function (doc) {
       }
       return;
     case 'task':
-    case 'target':
       return emit(doc.user, {});
+    case 'target':
```

Comment:
> Updates target to also use this value as their user property (for consistency), adds the owner property that saves the contact's uuid and updates medic/docs_by_replication_key so targets emit owner

You are updating the "owner" and "user" values in the documents to be consistent in that they will now _contain the same information_ (user contains `org.couchdb.user:abc` and owner contains guid of user contact).  I didn't pick the name `owner` that is from the FHIR standard and should mean "Responsible individual".  However, I did pick the name `user` -- let me know if you think that should be renamed in these.

Reply:
I think user is fine. An alternative would be to call it user_id, but it's too little of a change to consider.